frontend/src/routes/GymPage/index.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react'
+import GymStatus from '../../components/GymStatus'
+import GymKeyWith from '../../components/GymKeyWith'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components'
+import { GymTabContainer } from '../../components/Tabs'
+import { Icon } from 'antd-mobile'
+import ButtonComponent from '../../components/Button'
+
+const NavBarIcons = styled(Icon)`
+  &.am-icon-md {
+    width: 30px;
+    height: 30px;
+  }
+  margin-left: 0.5rem;
+  width: 30px;
+  height: 30px;
+`
+
+const GymBarContainer = styled.div`
+  position: sticky;
+  display: flex;
+  background-color: #fff;
+  align-items: center;
+  top: 0;
+`
+
+const DummyHistoryRow = styled.div`
+  justify-content: space-between;
+  text-align: center;
+  height: auto;
+  padding: 0.5rem;
+`
+
+export default function GymPage({ onLeftClick }: { onLeftClick?: () => void }) {
+  const gymHistory = [
+    {
+      gymStatus: true,
+      keyStatus: 'cooltelegram',
+      requesttime: 1643388954,
+      telegramHandle: 'cooltelegram',
+      userID: 'A0123456Z',
+    },
+    {
+      gymStatus: true,
+      keyStatus: 'anothernewdude',
+      requesttime: 1643389801,
+      telegramHandle: 'anothernewdude',
+      userID: 'A0234567M',
+    },
+  ]
+
+  const [currentTab, setCurrentTab] = useState<number>(1)
+  const sections = ['Gym', 'History']
+  const history = useHistory()
+
+  function TabPage(tabID: number) {
+    switch (tabID) {
+      case 1:
+        return (
+          <>
+            <GymStatus isOpen />
+            <GymKeyWith />
+            <div style={{ justifyContent: 'center', display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
+              <ButtonComponent state="primary" text="Key with me" onClick={() => undefined} />
+              <ButtonComponent state="primary" text="Return key" onClick={() => undefined} />
+              <ButtonComponent state="primary" text="Close Gym" onClick={() => undefined} />
+            </div>
+          </>
```

Comment:
```suggestion

const GymContainer = styled.div`
  padding: 50px 0;
`

export default function GymPage({ onLeftClick }: { onLeftClick?: () => void }) {
  const gymHistory = [
    {
      gymStatus: true,
      keyStatus: 'cooltelegram',
      requesttime: 1643388954,
      telegramHandle: 'cooltelegram',
      userID: 'A0123456Z',
    },
    {
      gymStatus: true,
      keyStatus: 'anothernewdude',
      requesttime: 1643389801,
      telegramHandle: 'anothernewdude',
      userID: 'A0234567M',
    },
  ]

  const [currentTab, setCurrentTab] = useState<number>(1)
  const sections = ['Gym', 'History']
  const history = useHistory()

  function TabPage(tabID: number) {
    switch (tabID) {
      case 1:
        return (
          <GymContainer>
            <GymStatus isOpen />
            <GymKeyWith />
            <div style={{ justifyContent: 'center', display: 'flex', flexDirection: 'column', alignItems: 'center' }}>
              <ButtonComponent state="primary" text="Key with me" onClick={() => undefined} />
              <ButtonComponent state="primary" text="Return key" onClick={() => undefined} />
              <ButtonComponent state="primary" text="Close Gym" onClick={() => undefined} />
            </div>
          </GymContainer>
```

Might be good to do something like this so that there is some spacing between the Tabs and the Gym Status

Reply:
Thanks ! c52221d5d27ae576679b8adeee965171545f0733